src/Helper/ValidatePostalCode.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Helper;
+
+use MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment;
+
+class ValidatePostalCode
+{
+    const VALIDATE_POSTAL_CODE_REGEX_NL = '/^[1-9][0-9]{3} ?[a-z]{2}$/i';
```

Comment:
```suggestion
    const VALIDATE_POSTAL_CODE_REGEX_NL = '/^[1-9]\d{3}\s?[a-z]{2}$/i';
```
Zo is het duidelijker dat het om een spatie gaat.